app/lib/importers/file_importer.rb

```diff
@@ -0,0 +1,40 @@
+class FileImporter
+  require "csv"
+
+  attr_reader :import_csv, :org_id, :number_imported, :failed_imports
```

Comment:
Could we rename `import_csv` to something like `csv_import_file` or `csv_import_filespec`?

Reply:
Or maybe we don't need `import` in the name at all, it's easy to infer.